src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="search-form" 
+      data-qa="big" 
+      action="/"
+    >
+      <img 
+        class="icon"
+        src="images\Search.svg" 
+        alt="searchGlass"
+      >
+      <input 
+        data-qa="keypress"
+        class="searchLine"
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form 
+      class="search-form small-form" 
```

Comment:
```suggestion
      class="search-form form--small" 
```
and fix in other places according to this example